example/web/public/.well-known/apple-app-site-association

```diff
@@ -0,0 +1,11 @@
+{
+    "applinks": {
+        "apps": [],
+        "details": [
+            {
+                "appID": "2732HQPVJP.com.splendo.kaluga.example",
```

Comment:
is it safe to share our teamID here?
It does make the example work, but only internally for us.
I'd rather put a `<DISCLAIMER>` instead, and add istructions in the README.
Something like `<your_apple_team_id_here>` and in README:
`To make the example work, you should make sure to change <your_apple_team_id_here> in apple-app-site-association file on your backend`

Reply:
This file is published online under a known public address, so I don't see the problem with putting it in a repo.

If this does not work for other using the example, then it'd be good to put something in the README none the less of course.

Reply:
yeah exactly, we discussed it also in chat: this file is already online, hence doesn't affect the development for Splendo employees in any way.
But since there is not much point adding it here, better to point out in a nice way to the external users of this library what needs to be changed and where.
Is more about documentation than security, actually.

Reply:
I mean, on iOS you must be in splendo team for this example to work.. correct?
That's not the same on Android, as you pointed out in the other comment.. we could share the debug key.

The question is:
- do we want to make it easy for external people to run the example out of the box (not sure how can be done on ios)
- OR: to we want to tell external users: use your own backend: here's the way to do it

imho

Reply:
As I wrote, the README should contain instructions. But we can leave ours by default.

Reply:
Indeed, we could use our file as a real example and put in README something generic as @posytive suggest